rs/src/parse/relations/set.rs

```diff
@@ -68,6 +68,19 @@ pub fn parse_set_rel(x: &substrait::SetRel, y: &mut context::Context) -> diagnos
         (SetOp::IntersectionMultiset, _) => Operation::Intersect,
         (SetOp::UnionDistinct, _) => Operation::Union,
         (SetOp::UnionAll, _) => Operation::Merge,
+        (SetOp::MinusPrimaryAll, _) | (SetOp::IntersectionMultisetAll, _) => {
+            diagnostic!(
+                y,
+                Warning,
+                NotYetImplemented,
+                "Variant {:?} not yet supported",
```

Comment:
should probably include word set in the message

Reply:
Yes, done 👍